src/test/resources/data/listOfValidAttachments.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+      "id": "123",
+      "name": "something.txt",
+      "contentType": "application/txt",
```

Comment:
Have you just invented new mime type? :D I guess you thought about `text/plain` :)

Reply:
I'm an engineer, I invent things 😆Okay i'll change it something that exists